frontend/src/tests/setup.ts

```
@@ -1,151 +1,151 @@
-	postMessage = mockWorker.postMessage;
-	addEventListener = mockWorker.addEventListener;
-	removeEventListener = mockWorker.removeEventListener;
-	terminate = mockWorker.terminate;
-	onmessage = null;
-	onmessageerror = null;
-	onerror = null;
-	onunhandledrejection = null;
-	onrejectionhandled = null;
-	dispatchEvent = mockWorker.dispatchEvent;
+  constructor() {
+    return mockWorker;
```

Biome (1.9.4)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor is ignored.

(lint/correctness/noConstructorReturn)